client/helpers_test.go

```diff
@@ -864,8 +864,8 @@ func TestApplyChangelistTargetsToMultipleRoles(t *testing.T) {
 	assert.False(t, ok, "no change to targets/level2, so metadata not created")
 }
 
-// ApplyTargets falls back to role that exists when adding or deleting a change
-func TestApplyChangelistTargetsFallbackRoles(t *testing.T) {
+// ApplyTargets does not fall back to role that exists when adding or deleting a change to a nonexistent delegation
+func TestApplyChangelistTargetsDoesNotFallbackRoles(t *testing.T) {
```

Comment:
Non-blocking nit:  If we're not doing fallback, I wonder if we should just leave the concept of fallback out of the comments and function names entirely?  It definitely makes sense to for the target to fail if the role doesn't exist (which is what happens here), and mentioning fallback might be somewhat confusing since we're not doing it anymore?